src/runtimes/rego/stack_pre.rs

```diff
@@ -1,8 +1,8 @@
-use anyhow::Result;
-
 use crate::policy_evaluator::RegoPolicyExecutionMode;
 use crate::policy_evaluator_builder::EpochDeadlines;
 
+use super::errors::{RegoRuntimeError, Result};
```

Comment:
nit: use crate and avoid super as per code conventions

Reply:
true! done inhttps://github.com/kubewarden/policy-evaluator/pull/424/commits/220620b9234e2a7016b2dd7bfd4d11cd5be0f8b4